sample/layman.map/internal_external_edge_cases.json

```diff
@@ -26,7 +26,7 @@
       "maxResolution": null,
```

Comment:
Please update documentation, that only layers named `l_<UUID>` in `layman` and `layman_wms` workspaces are considered internal:
- `Internal map layer` in models.md
- `file` parameter in POST Workspace Maps

Also add new point to changelog mentioning change in definition of internal map layer with link to models.md.